Formula/wine.rb

```diff
@@ -64,53 +51,421 @@ def require_universal_deps?
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--disable-lzma",
+                              "--without-x",
+                              "--with-jpeg-lib-dir=#{libexec}/lib",
+                              "--with-jpeg-include-dir=#{libexec}/include"
+        system "make", "install"
+      end
+
+      resource("little-cms2").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--with-jpeg=#{libexec}",
+                              "--with-tiff=#{libexec}"
+        system "make", "install"
+      end
+
+      resource("libpng").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static"
+        system "make", "install"
+      end
+
+      resource("freetype").stage do
+        # Enable sub-pixel rendering
+        inreplace "include/freetype/config/ftoption.h",
+                  "/* #define FT_CONFIG_OPTION_SUBPIXEL_RENDERING */",
+                  "#define FT_CONFIG_OPTION_SUBPIXEL_RENDERING"
+
+        system "./configure", "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--without-harfbuzz",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("libusb").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static"
+        system "make", "install"
+      end
+
+      resource("libusb-compat").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("webp").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--disable-gl",
+                              "--enable-libwebpmux",
+                              "--enable-libwebpdemux",
+                              "--enable-libwebpdecoder",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("fontconfig").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--with-add-fonts=/System/Library/Fonts,/Library/Fonts,~/Library/Fonts",
+                              "--localstatedir=#{var}",
```

Comment:
Stuffing the cache into the bottle feels sketchy but using a separate cache sounds correct.